src/Microsoft.SqlTools.ServiceLayer/DacFx/Contracts/DeploymentOptions.cs

```diff
@@ -193,6 +209,30 @@ public void InitializeNonBooleanTypeOptions(DacDeployOptions options)
             }
         }
 
+        /// <summary>
+        /// Preparing all object types which are considered as boolean options
+        /// </summary>
+        public void InitializeObjectTypesDictionary()
+        {
+            Type objectTypeEnum = typeof(ObjectType);
+            foreach (string name in Enum.GetNames(objectTypeEnum))
+            {
+                MemberInfo[] member = objectTypeEnum.GetMember(name);
+                MemberInfo info = member?.FirstOrDefault();
+                string displayName = info?.GetCustomAttribute<DisplayAttribute>().GetName();
+                if (displayName == null)
```

Comment:
Should we check for string.IsNullOrEmpty since that's what your logger message is?